src/odemis/acq/drift/__init__.py

```diff
@@ -44,23 +44,32 @@ class AnchoredEstimator(object):
     the global acquire, such as at the beginning of a line), and call .estimate()
     to measure the drift.
     """
-    def __init__(self, scanner, detector, region, dwell_time):
+
+    def __init__(self, scanner, detector, region, dwell_time, max_pixels=MAX_PIXELS, follow_drift=True):
         """
         scanner (Emitter)
         detector (Detector)
         region (4 floats)
         dwell_time (float)
+        #TOdo
+        max_pixels (int): the maximum number of pixels that the anchor region will use. If the region
+          acquired is large, then the scale is adjusted to maximum pixels
+        follow_drift (bool): If True, the anchor region is adjusted based on drift measure. It is useful with drift
+          compensation is based on updating anchor region. If False, the anchor
+          region is acquired in the same location. It is useful when drift compensation is based on
+          beam shift or stage movement
```

Comment:
I'd slightly adjust like the following:
`If True, the anchor region position is adjusted based on the drift measured. It is useful when drift
 compensation is done by adjusting the scanner settings. If False, the anchor
 region is fixed. It is useful when drift compensation is based on beam shift or stage movement.`